app/api/courses/courses.interface.ts

```diff
@@ -10,3 +10,11 @@ export interface CourseProps {
   created_at: string;
   duration: number;
 }
+
+export interface VideoProps {
+  id: string;
+  video_url: string;
+  title: string;
+  duration: number;
```

Comment:
If you are building an app for long term, probably it will be nice to document somewhere (or in the comment) the unit of measurements for duration